Dockerfile.dev

```diff
@@ -0,0 +1,22 @@
+FROM registry.fedoraproject.org/fedora:latest
+
+RUN mkdir /app
+WORKDIR /app
+
+# Some dependencies are only needed for aarch64
```

Comment:
Which dependencies are missing in the `aarch64` Fedora container? I'd like to look into that.

Reply:
Essentially I added the last line while trying to get it working on MAC
```
make git findutils file coreutils curl qemu-img tar util-linux python-docutils pkg-config
```
although I'm pretty sure not all are needed - I'll try to fine tune and document in the comment although I can't test right now (yet)

Reply:
If you need any testing I can provide.